qa/integration-tests/src/test/java/io/camunda/it/client/UserTaskQueryTest.java

```diff
@@ -444,16 +444,19 @@ public void shouldSortTasksByCreationDateASC() {
         .isLessThan(result.items().get(2).getCreationDate());
 
     // Assert First and Last Sort Value matches the first and last item
-    assertThat(result.page().firstSortValues())
+    // We need to make use of toString, such the test work with ES/OS
+    assertThat(result.page().firstSortValues().stream().map(Object::toString).toList())
```

Comment:
🙈 Sorry that you have to do this right now 😅 
🤷 I mean, this is a way of making sure the results are the same, type-wise, no matter which data layer is used 🤣